healthScore/templates/view_requests.html

```diff
@@ -0,0 +1,293 @@
+                    <label for="date" class="sr-only">Date:</label>
+                    <input type="date" class="form-control mb-2" id="date" name="date" value="{{ request.GET.date }}">
+                </div>
+                <div class="col-auto">
+                    <label for="record_status" class="sr-only">Request Status:</label>
+                    <select class="form-control mb-2" id="record_status" name="record_status"
+                            aria-label="Request Status">
+                        <option value="">Request Status</option>
+                        <option value="approved">Approved</option>
+                        <option value="rejected">Rejected</option>
+                        <option value="pending">Pending</option>
+                    </select>
+                </div>
+                <div class="col-auto">
+                    <button type="submit" class="btn btn-secondary mb-2">Filter</button>
+                </div>
+                <div class="col-auto">
+                    <a href="{% url 'view_requests' %}" class="btn btn-outline-secondary mb-2">Clear</a>
+                </div>
+            </form>
+
+        </div>
+    </div>
+
+    <!-- Data Display Section -->
+    <div class="row">
+        <div class="col-lg-12">
+            <table class="table table-striped my-4">
+                <thead class="thead-dark">
+                <tr>
+                    <th scope="col">Reason for Visit</th>
+                    <th scope="col">Visit Details</th>
+                    <th scope="col">Healthcare Worker</th>
+                    <th scope="col">Healthcare Facility</th>
+                    <th scope="col">Address</th>
+                    <th scope="col">Created Date</th>
+                    <th scope="col">Updated Date</th>
+                    <th scope="col">Status</th>
+                </tr>
+                </thead>
+                <tbody>
+                {% for request, details in zipped_details %}
+                    <tr>
+                        <td class="appointmentName"
+                            onclick="getProperties(`{{ forloop.counter }}|{{ request.appointment_properties }}|{{ request.appointment_name }}`)"
+                            data-toggle="modal" data-target="#myModal">{{ request.appointment_name }}</td>
+                        <td>{{ request.appointment_type }}</td>
+                        <td>{{ request.doctor_name }}</td>
+                        <td>{{ request.hospital_name }}</td>
+                        <td>{{ request.hospital_address }}</td>
+                        <td>{{ request.createdAt }}</td>
+                        <td>{{ request.updatedAt }}</td>
+                        <td class="{% if request.record_status == 'approved' %}text-success
+                            {% elif request.record_status == 'rejected' %}text-danger
+                            {% else %}text-dark{% endif %}">
+                            {{ request.record_status }}
```

Comment:
Could you please capitalize the first letter here?

Reply:
fixed